python/lsst/pipe/tasks/measurePsf.py

```diff
@@ -238,7 +237,7 @@ def __init__(self, schema=None, **kwargs):
         self.makeSubtask("psfDeterminer", schema=schema)
 
     @pipeBase.timeMethod
-    def run(self, exposure, sources, expId=0, matches=None):
+    def run(self, exposure, sources, expId=1, matches=None):
         """!Measure the PSF
```

Comment:
Fixed.  Didn't realize that there was a way to pass expId=0 in the pipeline, but I agree there should be a safety against 0.